packages/web-vite/src/pages/HighlightsPage.tsx

```
@@ -0,0 +1,224 @@
+        ) : (
+          <div className="highlights-list">
+            {highlights.map((highlight) => (
+              <div
+                key={highlight.id}
+                className="highlight-card"
+                onClick={() => handleHighlightClick(highlight)}
+                style={{
+                  borderLeftColor:
+                    HIGHLIGHT_COLORS[highlight.color] || HIGHLIGHT_COLORS.YELLOW,
+                }}
+              >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)